libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.h

```diff
@@ -30,11 +30,6 @@
 
 @class SFUserAccount;
 
-extern NSString * const kSyncManagerLocal;
```

Comment:
SFSyncTarget owns those constants. So an application could through a custom target decide to flag dirty records differently.